### ISSUE TITLE
memfs@2.8.0 breaks test case

### ISSUE THREAD

**Issue by aleung** [sapegin/mrm-core#14]
Package defines `"memfs": "^2.6.0"`.

If ignore `package-lock.json`, latest memfs@2.8.0 will be installed. Test cases fail with memfs@2.8.0:

```
 FAIL  src/files/__tests__/packageJson.spec.js
  ● packageJson() › should create package.json file

    expect(value).toMatchSnapshot()

    Received value does not match stored snapshot 1.

    - Snapshot
    + Received

      Object {
    +   "/": null,
        "/package.json": "{}",
      }

      at Object.it (src/files/__tests__/packageJson.spec.js:37:24)
          at new Promise (<anonymous>)
          at <anonymous>
      at process._tickCallback (../../../../../internal/process/next_tick.js:188:7)
```

**Comment by sapegin** [comment]
Looks like they've added directories support (arguably a braking change). I guess updating Jest snapshots should be enough. Feel free to send a PR ;-)

**Comment by aleung** [comment]
I'm trying to fix it.

**Comment by aleung** [comment]
The release version of memfs is confusing. I've raised an issue streamich/memfs#123 to them.
Temporary fix memfs version to 2.6.0 as a workaround.

**Comment by sapegin** [comment]
Another attempt to update: https://github.com/streamich/memfs/pull/174 ;-)
